### ISSUE TITLE
Did you achieve the same accuracy as the original crnn?

### ISSUE THREAD

**Issue by melody-rain** [meijieru/crnn.pytorch#5]
Hi can you achieve the same accuracy as the original crnn? And what's your accuracy?

Thanks.

**Comment by meijieru** [comment]
The provided pretrained model is converted from the origin one, so it has the same accuracy.

**Comment by melody-rain** [comment]
What I mean is if I train a model from scratch with your code with the same data of original crnn, could I get the same accuracy?

Have you train a model from scratch? thanks a lot.

**Comment by meijieru** [comment]
I haven't try to do this. However, the network architecture is the same. There may be some differences due to the strategy of training, but not big.

**Comment by liangshuang1993** [comment]
Hi, @meijieru Thanks for your work! I want to train this network from scratch. Currently I have generated about 10 million pictures using different fonts and backgrounds with different length. But the precision is not high enough. 

This is my trainging log.(from a model I saved before)
Test loss: 2.592091, accuray: 0.514375
[0/1000][5100/14023] Loss: 1.407239
[0/1000][5200/14023] Loss: 1.422566
[0/1000][5300/14023] Loss: 1.398798
[0/1000][5400/14023] Loss: 1.394613
[0/1000][5500/14023] Loss: 1.404841

The loss can't go any lower. And there's seems like to be over-fitting too.
So can you give me any guide? Is the dataset big or variable enough? or should I use fixed-length picture to train? 

Thanks a lot!!

**Comment by meijieru** [comment]
@liangshuang1993 Is it the same dataset trained for origin crnn model?

**Comment by liangshuang1993** [comment]
@meijieru Thanks for your reply. 

Maybe I should comment under this issue. The answer is no, I used a dataset generated by myself. About 300  Chinese and English characters. The dataset has variable length. 

So have you tried on Chinese character?  Do you think a dataset contains about 10 million pictures is enough for 300 classes? Thanks!

**Comment by meijieru** [comment]
Don't try on chinese and other dataset. It depends on the dataset.

**Comment by liangshuang1993** [comment]
Ok, so do you have any idea which network should I use to train Chinese? Thanks
